spec/syntax.md

```diff
@@ -462,9 +462,10 @@ except for surrogate code points U+D800 through U+DFFF.
 The characters `\` and `|` MUST be escaped as `\\` and `\|`.
 
 **_Unquoted_** literals have a much more restricted range that
```

Comment:
I think this is missing some information that could help avoid repeats of the conversation in this PR.

```suggestion
A _literal_ MAY appear **_unquoted_** when its content matches
the `unquoted` production. 

Similar to other productions in the message syntax,
the content of an _unquoted_ literal is identical to XML's [Nmtoken]
(https://www.w3.org/TR/xml/#NT-Nmtoken) except that specific characters
meaningful to other portions of the syntax are not allowed at the start. Specifically:

* A _literal_ MUST be quoted if it begins with a `:` (U+003A COLON).
* An _unquoted_ literal MUST NOT begin with a `-`  (U+002D HYPHEN MINUS)
unless that character is followed by a `.` (U+002E FULL STOP) or an ASCII digit.
This allows literals which might be used to represent negative numbers, such 
as `-42` or `-.345`, to appear unquoted.
```

Reply:
I'm fine with being more verbose about the description here, but the above suggestion is a bit problematic:
- The first paragraph is somewhat tautological, given that this is how practically all of the syntax is defined.
- I don't think we have any other productions that refer to or approximate XML Nmtoken. Our _name_ is rather close to XML Name, but its first-character rules are quire different.